### ISSUE TITLE
Not working wioth newer Home Assistant versions

### ISSUE THREAD

**Issue by rbrtwillis** [burnnat/ha-polar#16]
I'm running HA 2021.11.5 and couldn't install this with HACS. It says it's on a blacklist. Then I manually downloaded the files. After I restarted HA, the logs said the manifest.json was missing the versions attribute. I manually added it and restarted HA again. I added a basic configuration into the configuration.yaml. I registered with Polar as described and used my unique ID and Secret. When I restarted HA to load the config, it came up and said Loading Polar and it wouldn't go away. It blocked other things from loading. I guess this is too old and is incompatible with newer versions of HA. I might take a look at forking it and trying to fix it but that will need to wait until I have a lot more time. I'm not sure even where to start.  Would be great to get this working again. Thanks.

**Comment by woolf0007** [comment]
I would really appreciate it if you could do it i would love to use it!
My other idea would be that the polar can be connected to the strava but unfortunately the strava doesn’t work either.
Thx WLF

**Comment by PeterGamma** [comment]
"I'm running HA 2021.11.5 and couldn't install this with HACS". 

I wonder with what HA version it was running. I am ready to downgrade for this. It would be highly desirable to have a working version. I saw that there are a Fitbit HA integraton and a Garmin integration. The Garmin integration allows only to sync wellness parameters. The Polar here allows streaming of activity data. This is a feature I miss strongy.

**Comment by PeterGamma** [comment]
I already managed to record my Polar OH1 sensor data, found the right button to start recording, which was not so easy, and was able to load it up to Polar. It is making progress. Which was the last HA version ha-polar was executable?

**Comment by woolf0007** [comment]
I think not only the HA version is the problem. The polar use the new Oath , that this integration cant reach... if you can put the polar folder to the custom components manually, and change in the manifest.json the version number, you can see polar in the integration, but after that it is not reach the datas from polar. I gave it up few month ago...

**Comment by PeterGamma** [comment]
Thanks, woolf0007, but the "Activity Data from Strava directly into Home Assistant" project seems to be running:
 
https://github.com/woolf0007/ha_strava

**Comment by woolf0007** [comment]
Yes i know,, but iam not a coder, you should use this: https://github.com/craibo/ha_strava
Maybe it will be maitanence...

**Comment by Aohzan** [comment]
I did an updated integration for Polar if it can help https://github.com/Aohzan/hass-polar

**Comment by PeterGamma** [comment]
Thanks you very much, Matthieu, we are very happy about your update and highly appreciate your contribution.
